client/src/App.js

```diff
@@ -9,14 +10,17 @@ const Container = styled.div`
 `;
 
 const App = () => (
-  <Container>
-    <header>
-      <h1>shorty</h1>
-    </header>
-    <main>
-      <Shorties />
-    </main>
-  </Container>
+  <>
+    <GlobalStyle />
+    <Container>
+      <header>
+        <h1>shorty</h1>
```

Comment:
I don't find the article, but I remember that we should avoid doubled sematics like that. If you only have one semantic element in a section, you should remove the section. Just go with the `h1` remove the `header`.

Reply:
Thx, I agree with that.
In https://github.com/lmachens/shorty/pull/2/files#diff-700bde2449c4d70123c8cfdbca47c416849428e7856786d53d7a0b7c914d1208R23 I added `img` to `header`.